scripts/packaging/publish_release.py

```diff
@@ -129,6 +130,10 @@
                             print('Release upload failed due to connection error (remaining trials: %d)' % (4 - retryCount))
                         except requests.exceptions.HTTPError:
                             print('Release upload failed due to server error (remaining trials: %d)' % (4 - retryCount))
+                        except GithubException:
+                            print('Release upload failed due to GitHub error (remaining trials: %d): %s' % (4 - retryCount))
+                        except:
+                            print('Release upload failed due to unexpected error (remaining trials: %d)' % (4 - retryCount))
```

Comment:
```suggestion
                        except Exception as e:
                            print('Release upload failed due to unexpected error (remaining trials: %d)' % (4 - retryCount))
                            print('Error message', e)
```
What about printing the exception? Just to know what actually happened in case we end up in the general exception.